helix-term/src/ui/editor.rs

```diff
@@ -706,6 +769,315 @@ impl EditorView {
         );
     }
 
+    /// Render the sticky context
+    #[allow(clippy::too_many_arguments)]
+    pub fn render_sticky_context(
+        doc: &Document,
+        view: &View,
+        surface: &mut Surface,
+        context: &Option<Vec<StickyNode>>,
+        doc_annotations: &TextAnnotations,
+        line_decoration: &mut [Box<dyn LineDecoration + '_>],
+        translated_positions: &mut [TranslatedPosition],
+        theme: &Theme,
+    ) {
+        if let Some(context) = context {
```

Comment:
```suggestion
        let Some(context) = context else {
            return;
        }
```
Can do quick return here.

Reply:
Oh wow! This is something i actually didn't know you could do!
Thanks!